scripts/pi-hole/php/header.php

```diff
@@ -176,7 +176,7 @@ function pidofFTL()
 <head>
     <meta charset="utf-8">
     <meta http-equiv="Content-Security-Policy" content="default-src 'self' https://api.github.com; script-src 'self' 'unsafe-eval' 'unsafe-inline'; style-src 'self' 'unsafe-inline'">
-    <title>Pi-hole Admin Console</title>
+    <title>Pi-hole <?php echo gethostname() ?></title>
```

Comment:
```suggestion
    <title>Pi-hole on <?php echo gethostname() ?></title>
```

or 

```suggestion
    <title>Pi-hole - <?php echo gethostname() ?></title>
```

Reply:
If we're adding a seperator can we switch the order to
```<title><?php echo gethostname() ?> - Pi-hole</title>```?

Reply:
I'd prefer Pi-hole come first, so that is visible if the tab collapses. But not a make or break requirement from me.